pkg/skaffold/build/jib/jvm.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jib
+
+import (
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
+)
+
+var (
+	// JVMFound is true if a Java VM was found and works.
+	JVMFound bool
+)
+
+func init() {
```

Comment:
What are the benefits of doing this in an init function vs doing this with a `sync.Once`? Would this run for basically all skaffold invocations?

Reply:
It simplified the implementation slightly.  A package `init()` should only run whenever the package is loaded, but as I've discovered when putting a `panic()` into this function, we touch the Jib package early on:
```console
$ ../../../out/skaffold build -v debug
panic: resolveJvm() called

goroutine 1 [running]:
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/jib.resolveJVM(0x1400039a420)
	/Users/bdealwis/Projects/Skaffold/repo-skaffold/pkg/skaffold/build/jib/jvm.go:40 +0x38
github.com/GoogleContainerTools/skaffold/pkg/skaffold/build/jib.init.0()
	/Users/bdealwis/Projects/Skaffold/repo-skaffold/pkg/skaffold/build/jib/jvm.go:33 +0x20
```

I'll rewrite it to use a `sync.Once`.